docs/articles/beyond_basics/interactions.md

```diff
@@ -0,0 +1,41 @@
+# Interactions
+
+[Interactions](https://discord.com/developers/docs/interactions/receiving-and-responding#interactions) represent a user interacting with your bot.
+This can happen in different ways. The most prominent case is Application Commands ("slash commands") but also button presses or context menus.
+
+## Recieving interactions
+Discord offers two ways to recieve interactions: Gateway and HTTP webhooks.
+Currently DSharpPlus only supports the first one but there are plans to also integrate webhooks.
+To recieve an interaction you do not have to configure anything and simply register an EventHandler to the `InteractionCreated`event.
+
+Additional to that event we have some events that are filtered to provide some convienience:
+
+- Buttons and Selectmenues -> `ComponentInteractionCreated`
+- Modals -> `ModalSubmitted`
+- User or Message context menu -> `ContextMenuInteractionCreated`
+- Application Commands and autocompletion for those only in `InteractionCreated`
+
+## Handling an interaction
+The baseline is that every interaction has to be acknowledged in some kind in the first 3 seconds after recieving it.
+The possible responses are dependent on the type of interaction and the desired response.
+
+> [!Important]
+> The initial response has to decide if the repsonse should be ephemeral. You cant change this later
```

Comment:
missing an apostrophe in "can't", the wording also seems a bit off, but I can't think of a better way to word it :ioa:

Reply:
I dont know how to reword it either :when: